Services/Documents/DocumentService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Documents;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using System.IO;
+
+namespace Andculture.Sitefinity.Testing.Services.Documents
+{
+    public class DocumentService : ODataService<Document>
+    {
+        public DocumentService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/documents";
+        }
+
+        public IRestResponse Upload(Document model)
+        {
+            string documentTitle = "digital_marketing";
+            string documentExtension = ".pdf";
+            string docFullName = string.Concat(documentTitle, documentExtension);
+            string docFilePath = string.Concat(Directory.GetCurrentDirectory(), @"\Media\", documentTitle, documentExtension);
+            var jObject = JObject.FromObject(model);
+            jObject.Add("DirectUpload", true);
+
+            var url = Settings.BaseUrl + EndpointUrl;
```

Comment:
Suggestion: Similar to an earlier comment. This chunk of code is more like a request builder. I think there is an opportunity to abstract this out into a method that handles a lot of the boilerplate code. Then when you use it you're only providing the unique code (e.g. URL, HTTP method, headers collection, etc.).